hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -2521,4 +2521,32 @@
       filesystem semantics.
     </description>
   </property>
+
+  <property>
+    <name>ozone.om.metadata.cache.directory</name>
```

Comment:
This name also needs to be updated, current unit test was broken by this
> TestOzoneConfigurationFields>TestConfigurationFieldsBase.testCompareConfigurationClassAgainstXml:493 class org.apache.hadoop.ozone.OzoneConfigKeys class org.apache.hadoop.hdds.scm.ScmConfigKeys class org.apache.hadoop.ozone.om.OMConfigKeys class org.apache.hadoop.hdds.HddsConfigKeys class org.apache.hadoop.ozone.recon.ReconServerConfigKeys class org.apache.hadoop.ozone.s3.S3GatewayConfigKeys class org.apache.hadoop.hdds.scm.server.SCMHTTPServerConfig has 1 variables missing in ozone-default.xml Entries:   ozone.om.metadata.cache.directory.policy expected:<0> but was:<1>
[ERROR]   TestOzoneConfigurationFields>TestConfigurationFieldsBase.testCompareXmlAgainstConfigurationClass

Reply:
Fixed test failure